pcdsdevices/slits.py

```diff
@@ -62,6 +62,40 @@ def __init__(self, *args, nominal_aperture=5.0, **kwargs):
         super().__init__(*args, **kwargs)
         self.nominal_aperture.put(nominal_aperture)
 
+    def format_status_info(self, status_info):
+        """
+        Override status info handler to render the slits.
+
+        Display slits status info in the ipython terminal.
+
+        Parameters
+        ----------
+        status_info: dict
+            Nested dictionary. Each level has keys name, kind, and is_device.
+            If is_device is True, subdevice dictionaries may follow. Otherwise,
+            the only other key in the dictionary will be value.
+
+        Returns
+        -------
+        status: str
+            Formatted string with all relevant status information.
+        """
+        hutch = self.prefix.split(':')[0].upper()
+        stand = self.prefix.split(':')[1].upper()
+
+        x_width = get_status_value(status_info, 'xwidth', 'position')
+        y_width = get_status_value(status_info, 'ywidth', 'position')
+        x_center = get_status_value(status_info, 'xcenter', 'position')
+        y_center = get_status_value(status_info, 'ycenter', 'position')
+        w_units = get_status_value(status_info, 'ywidth', 'setpoint', 'units')
+        c_units = get_status_value(status_info, 'ycenter', 'setpoint', 'units')
+
+        return f"""\
+{hutch} Slit {self.name} on {stand}
```

Comment:
This will be malformed for the new slit PV naming in the FEE- I think we make note of this for a future follow-up since the current users aren't looking much at FEE slits

Example PV prefix in the FEE:
`SL1L0:POWER` -> `"SL1L0 Slit sl1l0 on POWER"`

Reply:
Aha I was just writing a question about this. Thank you.
This is because of lack of knowledge in my case... 

If I were to change it though, how would I know that it is a FEE pv, just by the prefix?

Reply:
I think making it more general is a really good call- leave it up to the viewer to know what SB2, XCS, SL1K0, etc. mean
Otherwise we have to pass in additional information (perhaps from happi metadata?)

Reply:
Ok I changed it in  - 24d752e to use the prefix.

Reply:
Including happi metadata is a good idea. (It's accessible by `obj.root.md`, if available, and is easily convertible into a dictionary)

Reply:
Ken, you're saying we can try/except/AttributeError to pull out the md dict and inspect it, if we expect that it may have useful information for us? I'm going to paste two example slit entries for reference, I think it's a good idea (and we can make it work by being more diligent with our database)
```
    "cxi_dg1_slits": {
        "_id": "cxi_dg1_slits",
        "active": true,
        "args": [
            "{{prefix}}"
        ],
        "beamline": "CXI",
        "creation": "Tue Feb 27 10:41:25 2018",
        "device_class": "pcdsdevices.device_types.Slits",
        "kwargs": {
            "name": "{{name}}"
        },
        "last_edit": "Thu Apr 12 14:40:08 2018",
        "macros": null,
        "name": "cxi_dg1_slits",
        "parent": null,
        "prefix": "CXI:DG1:JAWS",
        "screen": null,
        "stand": "DG1",
        "system": "beam control",
        "type": "pcdsdevices.happi.containers.Slits",
        "z": 1035.5
    },
```
    "sl1l0": {
        "_id": "sl1l0",
        "active": true,
        "args": [
            "{{prefix}}"
        ],
        "beamline": "LFE",
        "creation": "Tue Feb 25 17:55:08 2020",
        "detailed_screen": null,
        "device_class": "pcdsdevices.slits.PowerSlits",
        "documentation": null,
        "embedded_screen": null,
        "engineering_screen": null,
        "functional_group": "Slits",
        "kwargs": {
            "name": "{{name}}"
        },
        "last_edit": "Thu Feb 27 09:41:30 2020",
        "lightpath": true,
        "location_group": "Section 04",
        "macros": null,
        "name": "sl1l0",
        "parent": null,
        "prefix": "SL1L0:POWER",
        "stand": null,
        "system": null,
        "type": "pcdsdevices.happi.containers.LCLSItem",
        "z": 724.431
    },
```

Reply:
Yes, you correctly decrypted my overly terse comment. 😁 @teddyrendahl (I believe it was?) had some great forward-thinking to attach `md` to each instance, and could provide useful information in these printouts. Assuming it doesn't go against what the scientists are after, I suppose.

Reply:
Ok I like this, I can get the `beamline` and `stand`  form here which will be useful - I just need to figure out how exactly I do that... 
Looking at https://github.com/pcdshub/happi/pull/40 now

Reply:
I think you can use the `happi` cli to quickly check how this should look, e.g.

`happi load cxi_dg1_slits`
`happi load sl1l0`

Will each open ipython sessions that are useful for inspecting. The `md` attributes are `happi` containers.

Reply:
Can i assume that if i'm able to get the metadata it will have a `beamline` and `stand` attribute?
And can I also assume that I'll always have a `beamline` - it won't be None?

Reply:
So when I load it with happi:
```
In [1]: cxi_dg1_slits
Out[1]:
CXI Slit cxi_dg1_slits on DG1
(hg, vg): (+1.0954, +1.0976) [mm]
(ho, vo): (+0.0002, -0.0130) [mm]

```
And when I just create the device:
```
In [5]: l = LusiSlits(prefix='CXI:DG1:JAWS', name='s2')

In [6]: l
Out[6]:
Slit: CXI:DG1:JAWS
(hg, vg): (+1.0954, +1.0976) [mm]
(ho, vo): (+0.0002, -0.0130) [mm]
```
Is this fine?

Reply:
I think `beamline` is currently guaranteed while `stand` is not (for LCLS devices)

The printouts as you show here look pretty optimal to me